data_generator_tests.py

```diff
@@ -1,28 +1,94 @@
-
+# %% Load Annotations into dataframes
+import hourglass
+import imp
+imp.reload(hourglass)
```

Comment:
curious -- what does this do?

Reply:
force reloading modules allows updates to source file to be reflected in the notebook without needing to restart the notebook kernel. Otherwise module gets loaded once, and trying to import again will just use the module already in memory. This is useful since you wanna get anns in memory (takes 10-30 seconds) by running the first two cells once. But you're gonna be making lots of changes to data generator, so you'll be running the last cell many times and you don't wanna have to load data into memory every time.

Reply:
The example I just gave is more applicable to force reloading data generator on line 60. But I ended up making some changes to `hourglass.py` so that's why I was force reloading it.

Reply:
gotcha, thanks!